pkg/sql/logictest/testdata/logic_test/pg_builtins

```diff
@@ -158,3 +158,90 @@ query TT
+pg_class           relam                4294967205  7       0                         false
+pg_class           relfilenode          4294967205  8       0                         false
+pg_class           reltablespace        4294967205  9       0                         false
+pg_class           relpages             4294967205  10      0                         false
+pg_class           reltuples            4294967205  11      0                         false
+pg_class           relallvisible        4294967205  12      0                         false
+pg_class           reltoastrelid        4294967205  13      0                         false
+pg_class           relhasindex          4294967205  14      0                         false
+pg_class           relisshared          4294967205  15      0                         false
+pg_class           relpersistence       4294967205  16      0                         false
+pg_class           relistemp            4294967205  17      0                         false
+pg_class           relkind              4294967205  18      0                         false
+pg_class           relnatts             4294967205  19      0                         false
+pg_class           relchecks            4294967205  20      0                         false
+pg_class           relhasoids           4294967205  21      0                         false
+pg_class           relhaspkey           4294967205  22      0                         false
+pg_class           relhasrules          4294967205  23      0                         false
+pg_class           relhastriggers       4294967205  24      0                         false
+pg_class           relhassubclass       4294967205  25      0                         false
+pg_class           relfrozenxid         4294967205  26      0                         false
+pg_class           relacl               4294967205  27      0                         false
+pg_class           reloptions           4294967205  28      0                         false
+pg_class           relforcerowsecurity  4294967205  29      0                         false
+pg_class           relispartition       4294967205  30      0                         false
+pg_class           relispopulated       4294967205  31      0                         false
+pg_class           relreplident         4294967205  32      0                         false
+pg_class           relrewrite           4294967205  33      0                         false
+pg_class           relrowsecurity       4294967205  34      0                         false
+pg_class           relpartbound         4294967205  35      0                         false
+pg_class           relminmxid           4294967205  36      0                         false
+
+query B
+SELECT pg_column_is_updatable(oid, -1, true) FROM pg_class WHERE relname = 'is_updatable'
+----
+false
+
+# We make sure that the oid doesn't exists.
```

Comment:
nit: "Check that the oid does not exist." (doesn't exists is a typo!)

Reply:
Done.